doc/Addendum-NX-GZIP-for-PowerVM.txt

```diff
@@ -220,14 +220,14 @@ struct vas_tx_win_open_attr {
   Number of credits available in LPAR will be available in sysfs.
   Adding the following sysfs interfaces to the user-space:
 
-/sys/kernel/vas/VasCaps/VDefGzip:    (Called default GZIP capabilities)
-	avail_lpar_creds  target_lpar_creds  used_lpar_creds
+/sys/devices/virtual/misc/vas/vas0/gzip/default_capabilities/    (Called default GZIP capabilities)
+	nr_total_credits nr_used_credits
 
   On our development system with 16 cores and dedicated: these entry
-  values are 320 320 0 respectively
+  values are 320 320 respectively

```

Comment:
Does that mean nr_used_credits starts at 320? Is that correct?

Reply:
Nice catch. No, it's not correct. I should've deleted the second `320` instead. I'll fix it.

Reply:
Fixed.